packages/swingset-liveslots/test/liveslots-real-gc.test.js

```diff
@@ -26,7 +26,21 @@ const gcAndFinalize = makeGcAndFinalize(engineGC);
 // inconsistent GC behavior under Node.js and AVA with tests running
 // in parallel, so we mark them all with test.serial()
 
-test.serial('liveslots retains pending exported promise', async t => {
+// some tests may be flaky, so we use retry logic
+function testSerialTry(title, implementation) {
```

Comment:
that it uses `t.try` isn't a concern of the caller, so I'd remove that from the name. But it should convey that it **re**-tries.

I expect this to be useful in other tests. WDYT of putting in in @agoric/internal? It shoudl have some jsdoc in that case.

```suggestion
/**
  * Run the test serially, and if it fails retry once.
  * @param {string} title
  * @param (Function) predicate
  */
function testSerialRetry(title, implementation) {
```